### PR TITLE
closes #73 admin can not modify user info

### LINKED ISSUE #73: Admin cannot modify users
As a logged in Admin
I can modify my account data

But I cannot modify any other user's account data

### PR DESCRIPTION
**Pull request by Tman22**
Users can edit their own information, but can't access other users information. Also added new migrations to user: Email, street, city, state, zipcode.

### CODE REVIEW

File: test/factories/factories.rb [hunk 1]
```diff
@@ -27,6 +27,11 @@
     username
     password "password"
     role 0
+    email_address
+    street_address "123 Maple Drive"
+    city "Denver"
+    state "CO"
+    zipcode 80231
```
Comment (ToniRib) [inline test/factories/factories.rb]: I don't think we need to do this with a zipcode, it doesn't make any sense and makes it more confusing to read.

File: db/seeds.rb [hunk 2]
```diff
@@ -126,7 +131,12 @@ def create_users
                    last_name: last_name,
                    username: "#{first_name.downcase}_user",
                    password: "password",
-                   role: 0)
+                   role: 0,
+                   email_address: "#{first_name.downcase}_user@gmail.com",
+                   street_address: "123 Maple Drive",
+                   city: "Denver",
+                   state: "CO",
+                   zipcode: 80231)
```
Comment (ToniRib) [inline db/seeds.rb]: I don't think we need to do this with a zipcode, it doesn't make any sense and makes it more confusing to read.

File: db/seeds.rb [hunk 3]
```diff
@@ -35,7 +35,12 @@ def create_artists
                    last_name: last_name,
                    username: "#{first_name.downcase}_artist",
                    password: "password",
-                   role: 1)
+                   role: 1,
+                   email_address: "#{first_name.downcase}_artist@gmail.com",
+                   street_address: "123 Maple Drive",
+                   city: "Denver",
+                   state: "CO",
+                   zipcode: 80231)
```
Comment (ToniRib) [inline db/seeds.rb]: I don't think we need to do this with a zipcode, it doesn't make any sense and makes it more confusing to read.

**Comment by ToniRib** [comment]
I like these changes, what do you think @martensonbj?

**Comment by ToniRib** [comment]
Everyone cool with us skipping these hound violations with the zipcode?

**Comment by martensonbj** [comment]
Love it. We could possibly make the zip code a string if we want to get around the Hound complaints, but I'm also OK with ignoring that bit.

**Comment by Tman22** [comment]
I agree, we should skip that hound violation.

**Pull request merged**